Source/WebKit/NetworkProcess/storage/CacheStorageDiskStore.cpp

```diff
@@ -243,8 +244,13 @@ static std::optional<RecordHeader> decodeRecordHeader(Span<const uint8_t> header
     if (!responseHeadersGuard)
         return std::nullopt;
 
-    WebCore::ResourceResponse response;
-    if (!WebCore::ResourceResponse::decode(decoder, response))
+    std::optional<bool> isNull;
+    decoder >> isNull;
+    if (!isNull || *isNull)
+        return std::nullopt;
+
+    auto responseData = ResourceResponseBase::CrossThreadData::decode(decoder);
```

Comment:
Should we bump the version number?

Reply:
I think we don't need to as the disk storage format does not change.

Reply:
OK, please make sure this is actually the case by trying to load past stored data with the new decoder.
I guess this might explain the isNull encoding?
If so, maybe add a FIXME that clearly says to remove isNull when we bump the version number

Reply:
Yes, isNull is encoded and decoded to keep the same disk storage format.